main.elm

```diff
@@ -30,15 +30,23 @@ type alias Coord =
     ( Int, Int )
 
 
+{-| Dated values
+-}
+type alias Dated a =
```

Comment:
Yeah I could not find the doc about that, nor an example of it. I was not sure it was doable... But that's what I wanted to do :3